scripts/dev_docs.sh

```diff
@@ -88,9 +88,12 @@ module.exports = {
     ],
     "nav": {
       "structure": [
-        "nav-elastic-developer-guide",
-        "nav-kibana-team",
-        "nav-kibana-dev"
+        {
+          collection: 'Elastic Developer Guide',
+          navs: ['nav-elastic-developer-guide'],
+        },
+        { collection: 'Kibana Team', navs: ['nav-kibana-team'] },
+        { collection: 'Kibana Developer Guide', navs: ['nav-kibana-dev'] },
```

Comment:
For some reason I was getting errors when these where just strings... Not sure whether it was necessarily this but I updated it and it worked so 🤷🏻